lib/ai/tools.ts

```
@@ -0,0 +1,286 @@
+  execute: async ({ data, format, columns }) => {
+    if (!data || data.length === 0) {
+      return { error: 'No data provided' };
+    }
+
+    const keys = columns || Object.keys(data[0]);
+
+    switch (format) {
+      case 'json':
+        const jsonData = columns 
+          ? data.map(row => keys.reduce((acc, key) => ({ ...acc, [key]: row[key] }), {}))
+          : data;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)